Source/WebCore/page/FrameView.cpp

```diff
@@ -5842,6 +5842,15 @@ OverscrollBehavior FrameView::verticalOverscrollBehavior()  const
         return scrollingObject->style().overscrollBehaviorY();
     return OverscrollBehavior::Auto;
 }
+
+bool FrameView::isVisibleToHitTesting() const
+{
+    bool isVisibleToHitTest = false;
```

Comment:
This should default to true, since the following few lines only apply to subframes, and the root frame is always visible.

Reply:
Ah ok I see. Fixed.